scripts/config.js

```diff
@@ -43,6 +43,11 @@ export const packages = [
     packageDir: 'packages/query-sync-storage-persister',
     entries: ['main', 'module', 'types'],
   },
+  {
+    name: '@tanstack/qwik-query',
+    packageDir: 'packages/qwik-query',
+    entries: ['main', 'module', 'types'],
+  },
```

Comment:
@gioboa I added the necessary wiring in our config to release the package. Note that your `package.json` has no `module` entry. If that's intentional, please remove it from here. If it's not, please add it.

Reply:
I added it. thanks 👍